assets/css/translation.css

```diff
@@ -126,3 +126,67 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#localeSelectContainer {
+     width: 35%;
+}
+
+#primaryLocaleGlobe {
+     padding: 5px 5px 0 0;
+}
+
+#translateDoc {
+     margin-top: 1em;
+}
+
+#fileDropBackdrop {
+     z-index: 1000;
+}
+
+#dropDocumentContainer {
+     left: 35%;
+     top: 35%;
+}
+
+#fileDropMask {
+     opacity: 0;
+     z-index: 10000;
+}
+
+#uploadErrorContainer {
+     margin-top: 1em;
+     min-height: 50px;
+}
+
+#progressContainer {
+     margin-bottom: 0;
+}
+
+#fileLoading {
+     left: 50%;
+     top: 50%;
+}
+
+#morphAnalyzerOutput {
+     border-spacing: 0;
+}
+
+#selectLanguage {
```

Comment:
The element associated with this id isn't a language selector?

Reply:
hi @sushain97 sorry for late.How about `italicstyleRepresentation`

Reply:
id's should describe the element being styled, not how to style it. classes should describe how to style.

Reply:
@sushain97 here there is no element being styled in the `i` tag. `<i class="fa fa-globe fa-2x pull-left localeGlobe" id="       "></i>`. Sorry if i am wrong. Do you Mind suggesting any names?